pitop/pma/adc_base.py

```diff
@@ -57,3 +72,7 @@ def __read_peak(self, channel):
     # read 16 bits register
     def __read_register(self, read_address):
         return self.__adc_device.read_unsigned_word(read_address, little_endian=True)
+
+    @property
+    def value(self):
+        pass
```

Comment:
I am guessing that this gets overridden? Should we have `NotImplementedError` here?